about.html

```diff
@@ -0,0 +1,269 @@
+              href="https://www.linkedin.com/in/khitermed/"
+              target="_blank"
+              rel="noopener noreferrer"
+              >MyPage</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-white-50" href="#">Korean</a>
+          </li>
+        </ul>
+      </nav>
+      <!-- the white navbar for desktop version -->
+      <nav class="white-nav shadow p-3 mb-3 bg-body rounded only-desktop">
+        <ul class="nav container justify-content-around align-items-center">
+          <li class="nav-item me-5">
+            <a class="nav-link" href="./index.html"
+              ><img
+                src="img/TNW_Logo.png"
+                class="only-desktop me-5"
+                alt="TNW icon"
+            /></a>
+          </li>
+          <li class="nav-item ms-5">
+            <a class="nav-link secondary-color" href="./about.html">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link secondary-color" href="#">Program</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link secondary-color" href="#">Join</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link secondary-color" href="#">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link secondary-color" href="#">News</a>
+          </li>
+          <li class="nav-item ms-5">
+            <button type="button" class="get-tickets-btn">
+              Get tickets for TNW
+            </button>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section id="about-tnw">
+        <img
+          src="img/menu_black_48dp.svg"
+          alt="menu icon"
+          class="menu-icon only-mobile"
+          id="menu-btn"
+        />
+        <!-- Implementing the Mobile Menu -->
+        <div id="mobile-menu" class="only-mobile hidden">
+          <div class="pb-5"><a href="#" class="close" id="close-btn"></a></div>
+          <ul class="d-block pt-5">
+            <li><a href="./index.html" class="menu-link">Main Page</a></li>
+            <li><a href="#logo-section" class="menu-link">Our Logo</a></li>
+            <li>
+              <a href="#previous-conference" class="menu-link"
+                >Latest Conferences</a
+              >
+            </li>
+            <li><a href="#partners" class="menu-link">Sponsor</a></li>
+          </ul>
+        </div>
+        <div class="wrapper">
+          <p class="mx-4 primary-color h6 pt-1 text-center only-mobile">
+            "Get insights from industry pioneers"
+          </p>
+          <h1 class="mx-4 mt-2 display-5 primary-color text-center">
+            The Europe's leading tech festival!, TNW conference, NOW.
+          </h1>
+          <div
+            class="card m-4 mt-2 border-3 border-white bg-light"
+            id="intro-card"
+          >
+            <div class="card-body text-black-50">
```

Comment:
- [ ] [REQUIRED] On the mobile homepage, let's make sure that the picture is aligned to the center just as we have it provided in the behance design.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-23 00-01-49](https://user-images.githubusercontent.com/58692590/164816327-93a8be07-f769-4bc1-8bae-ccf97ee58c92.png) | ![Screenshot from 2022-04-23 00-02-06](https://user-images.githubusercontent.com/58692590/164816338-fcdfca44-3989-4446-8a92-19bef415bd71.png)

Reply:
It's done perfectly now!🔥.